Include SNIa rate plots to the pipeline

Fonotec opened the pull request:
This adds a couple of SNIa rate plots as a function of stellar mass and SFR to the pipeline. I did not add the observational data but that will come later this week.

bwvdnbro commented:
Note that this requires the latest version of the `velociraptor` module.

EvgeniiChaikin commented on colibre/auto_plotter/snia.yml:
```diff
@@ -0,0 +1,247 @@
+    number_of_bins: 30
+    start:
+      value: 1e-3
+      units: "1/(1e9*year)" 
```
```suggestion
      units: "1/gigayear"
```

EvgeniiChaikin commented on colibre/auto_plotter/snia.yml:
```diff
@@ -0,0 +1,247 @@
+      units: "1/(1e9*year)" 
+    end:
+      value: 1e1
+      units: "1/(1e9*year)" 
```
```suggestion
      units: "1/gigayear"
```

EvgeniiChaikin commented on colibre/auto_plotter/snia.yml:
```diff
@@ -0,0 +1,247 @@
+  legend_loc: "upper left"
+  x:
+    quantity: "derived_quantities.specific_sfr_gas_50_kpc"
+    units: "1/(1e9*year)"
```
```suggestion
    units: "1/gigayear"
```
Otherwise the units look strange

![Screenshot from 2022-09-27 14-21-10](https://user-images.githubusercontent.com/20153933/192525790-3349baa6-1950-4ee1-94c6-9982b8bfdd6a.png)

EvgeniiChaikin reviewed (changes requested):
@Fonotec Thanks! All looks good! 

I have one small suggestion in the code regarding the units, and two more general comments:

- Please specify `show_on_webpage: false` in `metadata` of all plots that use quantities computed in apertures different from 50 kpc. 
- I would be a bit more explicit and change the name of the file from **snia.yml** to **snia_rates.yml**

Also, are you planning to add observational data in this PR or in another PR?

Fonotec commented:
I am planning to add observational data in another PR. But this is the version with your suggestions. I also updated a few typos and removed that it is just from the centrals. I probably have to change a few more things but I think it is good to already have a working version with the main plots.

EvgeniiChaikin commented on colibre/auto_plotter/snia_rates.yml:
```diff
@@ -0,0 +1,325 @@
+  selection_mask: "derived_quantities.is_active_50_kpc"
+  x:
+    quantity: "derived_quantities.gas_o_abundance_fromz_avglin_50_kpc"
+    units: dimensionless 
```
Please insert the following

```suggestion
    units: dimensionless 
    log: false
```
otherwise there will be a log scale applied to log (O/H)

EvgeniiChaikin commented on colibre/auto_plotter/snia_rates.yml:
```diff
@@ -0,0 +1,325 @@
+    quantity: "derived_quantities.specific_sfr_gas_50_kpc"
+    units: "1/gigayear"
+    start: 1e-3
+    end: 1e1
```
```suggestion
    end: 3e1
```

EvgeniiChaikin commented on colibre/auto_plotter/snia_rates.yml:
```diff
@@ -0,0 +1,325 @@
+      value: 1e-3
+      units: "1/gigayear"
+    end:
+      value: 1e1
```
```suggestion
      value: 3e1
```

EvgeniiChaikin reviewed (approved):
Thank you @Fonotec!

I added three last comments to the code. Otherwise, it is ready to be merged.
